isamples_metadata/SmithsonianTransformer.py

```diff
@@ -185,8 +191,8 @@ def produced_by_feature_of_interest(self) -> str:
         return Transformer.NOT_PROVIDED
 
     def _add_to_responsibilities(self, source_label: str, dict_label: str, responsibilities: list[dict[str, str]]):
-        value = self.source_record[source_label]
-        if len(value) > 0:
+        value = self.source_record.get(source_label)
+        if value is not None and len(value) > 0:
```

Comment:
These were causing crashes for me running locally -- not sure why we hadn't seen them before, but I went ahead and fixed it since it was straightforward and safe.